Formula/sickchill.rb

```diff
@@ -0,0 +1,417 @@
+class Sickchill < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automatic Video Library Manager for TV Shows"
+  homepage "https://sickchill.github.io"
+  url "https://files.pythonhosted.org/packages/ba/75/d27b26bdac593423026a8d606551de134f745fb36ec991ff0a70d4dfb2a0/sickchill-2023.5.30.tar.gz"
+  sha256 "be49f0d6c01f479cb0f066e88582c4971b683a11d50149ffe81b1bdc4447c2ab"
+  license "GPL-3.0-or-later"
+
+  depends_on "rust" => :build
+  depends_on "cffi"
+  depends_on "pycparser"
+  depends_on "python@3.11"
+  depends_on "pyyaml"
```

Comment:
```suggestion
  # `pkg-config`, `rust`, and `openssl@3` are for cryptography.
  depends_on "pkg-config" => :build
  depends_on "rust" => :build
  depends_on "cffi"
  depends_on "openssl@3"
  depends_on "pycparser"
  depends_on "python@3.11"
  depends_on "pyyaml"
```
This should hopefully fix the `cryptography` build.